### ISSUE TITLE
Ratings are not scraped when I scrape a TV show

### ISSUE THREAD

**Issue by hewwra** [Komet/MediaElch#837]
**Scrapers that don't work:**

 - [X] IMDb
 - [ ] OFDb
 - [ ] TMDb
 - [ ] TMDbConcerts
 - [X] TheTvDb
 - [ ] TvTunes
 - [ ] AEBN
 - [ ] AdultDvDEmpire
 - [ ] HotMovies
 - [ ] VideoBuster

**Details which are wrong / not loaded:**
If I scrape a new TV show the ratings are not saved to tvshow.nfo regardless if I choose tvdb or imdb. It does work on the episode level but not on the TV show level.
This is how it looks like in my tvshow.nfo:

`<ratings>`
             `<rating` default="true" `name="default">`
             `<value>0</value>`
             `<votes>0</votes>`
        `</rating>`
 `</ratings>`

I dont know if it matters but  default="true" is displayed after the name variable in the episode nfo and it should probably be in the same order in the tvshow.nfo

**MediaElch Version:**
 - [X] 2.6.3-dev (nightly)
 - [ ] 2.6.2 stable

**Operating System:**
 - [X] Windows
 - [ ] macOS
 - [ ] Linux (distribution: ___)

**Additional context:**

**Comment by bugwelle** [comment]
Hi,

sorry it took me so long to answer. I had a lot to do in the last month and didn't get to debug this issue.

So, I tried to reproduce this issue without any luck. TheTvDb ratings work well for the shows I've tested (Family Guy, American Dad, Gravity Falls, Lost, Scrubs, Simpsons and more). What show does not work? :-)

However, I actually found a bug of mine in which I didn't store IMDb ratings correctly: 

https://github.com/Komet/MediaElch/blob/05de1c9b0f9d354cefbffee44acfcd9a652659e7/src/scrapers/tv_show/TheTvDb.cpp#L311

That will be fixed by: #843

> I dont know if it matters but default="true" is displayed after the name variable in the episode nfo and it should probably be in the same order in the tvshow.nfo

That isn't an issue :-)
The order of tags is actually random (due to how Qt implements `QHashMap`).
